packages/at_chops/lib/src/at_chops_impl.dart

```diff
@@ -361,4 +361,10 @@ class AtChopsImpl extends AtChops {
       throw InvalidDataException('Unrecognized type of data: $data');
     }
   }
+
+  @override
+  String readPublicKey(String publicKeyId) {
+    // This method is implemented only for extensions of AtChops that use secure element for private keys
```

Comment:
Again this is just one example use (the main example for sure) but I could choose to implement this in some other way also

Reply:
please check whether my latest docs are fine.